include/ogdf/basic/Logger.h

```diff
@@ -139,8 +149,16 @@ class Logger {
 	}
 
 	//! stream for logging-output (local)
-	std::ostream& lout(Level level = Level::Default) const {
-		return is_lout(level) ? *world : nirvana;
+	std::ostream& lout(Level level = Level::Default, bool indent = true) const {
+		if (is_lout(level)) {
+			if (indent) {
+				return *world << indenttabs();
```

Comment:
I think you can just get rid of `indenttabs()` and use the string-constructor instead: `std::string(m_by, '\t')`.
... And then squash that as well as the "Minor Improvements to Logger::Indent" into the first commit and I'm happy. :D

Reply:
Ah, didn't know that one. Thanks, fixed!